Change default menu name so that it does not get confused with
xscreensaver preferences.

$OpenBSD: patch-data_gnome-screensaver-preferences_desktop_in,v 1.2 2009/10/14 09:55:24 ajacoutot Exp $
--- data/gnome-screensaver-preferences.desktop.in.orig	Wed Oct 14 11:33:21 2009
+++ data/gnome-screensaver-preferences.desktop.in	Wed Oct 14 11:33:33 2009
@@ -1,5 +1,5 @@
 [Desktop Entry]
-_Name=Screensaver
+_Name=Screensaver (GNOME)
 _Comment=Set your screensaver preferences
 Icon=preferences-desktop-screensaver
 Exec=gnome-screensaver-preferences
